src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/DumpTask.groovy

```diff
@@ -22,6 +22,7 @@ class DumpTask extends AbstractTask {
     @TaskAction
     void dump() {
         def params = buildLibertyMap(project);
+        copyConfigFiles()
```

Comment:
I actually don't agree that this should be done.  Perhaps Maven is mistaken too. If the user wants a dump of the current runner server, if we copy the files, that could change the server they want to dump.  I created issue https://github.ibm.com/was-liberty/ecosystem/issues/144 as a follow-up issue.  I think remove this for now.